task/t1.py

```diff
@@ -0,0 +1,25 @@
+import os
+from utils.read_files import read_files
+
+
+file_values = read_files()
+
+temperatures = {}
+for file_value in file_values:
+    if file_value and file_value[1]:
+        temperatures[file_value[0]] = float(file_value[1])
```

Comment:
min temp and max temp are different columns!!!!

Reply:
done

Reply:
it is not done! you are still the using same one column